convert2rhel/actions/pre_ponr_changes/kernel_modules.py

```diff
@@ -71,10 +71,7 @@ def _get_rhel_supported_kmods(self):
             "--releasever=%s" % system_info.releasever,
             "--setopt=*.skip_if_unavailable=False",
         ]
-        basecmd = [
-            "repoquery",
-            "--releasever=%s" % system_info.releasever,
-        ]
+        basecmd = ["repoquery", "--releasever=%s" % system_info.releasever, "--setopt=exclude="]
```

Comment:
Could we add a comment here as well about setopt? Maybe just linking to the Jira issue. More comments the better for future maintainability IMO